tests/conftest.py

```diff
@@ -81,6 +82,54 @@ def clear():
     _Global._transpiler = None
 
 
+@pytest.fixture
+def star_connectivity():
+    def _star_connectivity(names=["q0", "q1", "q2", "q3", "q4"], middle_qubit_idx=2):
+        chip = nx.Graph()
+        chip.add_nodes_from(names)
+        graph_list = [
+            (names[i], names[middle_qubit_idx])
+            for i in range(len(names))
+            if i != middle_qubit_idx
+        ]
+        chip.add_edges_from(graph_list)
+        return chip
+
+    return _star_connectivity
+
+
+@pytest.fixture
+def grid_connectivity():
+    def _grid_connectivity(names=["q0", "q1", "q2", "q3", "q4"]):
+        chip = nx.Graph()
+        chip.add_nodes_from(names)
+        graph_list = [
+            (names[0], names[1]),
+            (names[1], names[2]),
+            (names[2], names[3]),
+            (names[3], names[0]),
+            (names[0], names[4]),
+        ]
+        chip.add_edges_from(graph_list)
+        return chip
+
+    return _grid_connectivity
+
+
+@pytest.fixture
+def line_connectivity():
+    def _line_connectivity(n, names=None):
```

Comment:
I'd say that is not worth making it a fixture, to return a function.

However, what maybe you don't know is that you can natively parametrize the fixtures during their invocation :)
```py
@pytest.fixture
def fixt(request):
    marker = request.node.get_closest_marker("fixt_data")
    return None if marker is None else marker.args[0]


@pytest.mark.fixt_data(42)
def test_fixt(fixt):
    assert fixt == 42
```
https://docs.pytest.org/en/stable/how-to/fixtures.html#using-markers-to-pass-data-to-fixtures